### PR TITLE
Create snaps E2E test for update function

### PR DESCRIPTION
**Pull request by bowensanders**
This creates a new e2e test to verify that the update function and UI are operating properly.

This was largely based on earlier tests I wrote for other snaps functions.

### CODE REVIEW

File: test/e2e/snaps/test-snap-notification.spec.js [hunk 1]
```diff
@@ -98,12 +98,9 @@ describe('Test Snap Notification', function () {
 
         // look for the correct text in notifications (via xpath)
         const notificationResultMessage = await driver.findElement(
-          '.notifications__item__details__message',
-        );
-        assert.equal(
-          await notificationResultMessage.getText(),
-          'TEST INAPP NOTIFICATION',
+          './/*[@id="app-content"]/div/div[3]/div/div[2]/div[2]/div[2]/div/div[2]/div[1]/div[2]/div[2]/span[2]-settings-card__version',
```
Comment (rekmarks) [inline test/e2e/snaps/test-snap-notification.spec.js:101]: This looks cursed, what is even going on here? 😅

**Comment by bowensanders** [comment]
This should NOW be ready for review again :)

File: test/e2e/snaps/test-snap-update.spec.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+        // try to click on the snaps item
+        await driver.clickElement({
+          text: 'Snaps',
+          tag: 'div',
+        });
+        await driver.delay(500);
+
+        // look for the correct version text
+        const notificationResultMessage = await driver.findElement(
```
Comment (FrederikBolding) [inline test/e2e/snaps/test-snap-update.spec.js:118]: Could this be named more aptly?

**Review by FrederikBolding — DISMISSED** [review]
LGTM, one small question about naming.

File: test/e2e/snaps/test-snap-update.spec.js (hunk 2 above)
Comment (bowensanders) [reply test/e2e/snaps/test-snap-update.spec.js]: renamed this versionResult now

**Review by FrederikBolding — APPROVED** [review]

**Review by hmalik88 — APPROVED** [review]

**Pull request merged**